.github/workflows/python.yaml

```diff
@@ -165,7 +165,7 @@ jobs:
           pip install "xgboost_ray<0.1.14" "protobuf<4" "sqlalchemy<2"
         fi
 
-        pip install -e ".[dev,extra]"
```

Comment:
Why do you remove the double quotes?

Reply:
when path is "s3", function "get_fs" requires "s3". So I rewrite this line "pip install -e .[dev,extra,aws]"

Reply:
But you can keep the double quotes, I don't think it should be removed.